qa/integration-tests/src/test/java/io/camunda/zeebe/it/gateway/GatewayGrpcHealthIT.java

```diff
@@ -10,36 +10,32 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 import io.camunda.zeebe.gateway.protocol.GatewayGrpc;
-import io.camunda.zeebe.qa.util.testcontainers.ZeebeTestContainerDefaults;
+import io.camunda.zeebe.qa.util.cluster.TestStandaloneGateway;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration.TestZeebe;
 import io.grpc.ManagedChannel;
 import io.grpc.health.v1.HealthCheckRequest;
 import io.grpc.health.v1.HealthCheckResponse;
 import io.grpc.health.v1.HealthCheckResponse.ServingStatus;
 import io.grpc.health.v1.HealthGrpc;
 import io.grpc.netty.NettyChannelBuilder;
 import io.grpc.protobuf.services.HealthStatusManager;
-import io.zeebe.containers.ZeebeGatewayContainer;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.ValueSource;
-import org.testcontainers.junit.jupiter.Container;
-import org.testcontainers.junit.jupiter.Testcontainers;
 
-@Testcontainers
+@ZeebeIntegration
 class GatewayGrpcHealthIT {
 
-  @Container
-  private final ZeebeGatewayContainer zeebeGatewayContainer =
-      new ZeebeGatewayContainer(ZeebeTestContainerDefaults.defaultTestImage())
-          .withoutTopologyCheck();
+  // can never be ready nor await the complete topology without a gateway
```

Comment:
without a broker, right?
```suggestion
  // can never be ready nor await the complete topology without a broker
```

Reply:
will apply to merge, let me know if I got this wrong so we can fix it after